linter_exclusions.yml

```diff
@@ -260,6 +260,9 @@ aks create:
       rule_exclusions:
       - option_length_too_long
     enable_encryption_at_host:
+      rule_exclusions:
```

Comment:
You'd better to add this exclusion in https://github.com/Azure/azure-cli/blob/e2465ce2ae1bfedbf5cc60ba0075139613d96dde/src/azure-cli/azure/cli/command_modules/acs/linter_exclusions.yml as this file is for general exclusion, but not for specific service.